### ISSUE TITLE
Add Active Effect for a minimum possible roll for skill checks

### ISSUE THREAD

**Issue by liam-edwards** [foundryvtt/dnd5e#3592]
I would like to see a Active Effect added that allows you to set the minimum possible roll on the d20 for specific skills. This is similar to how a Rogue's Reliable Talent works, setting their minimum possible in all proficient skills to a 10. What I'm requesting is this to be more configurable.

How I imagine this working is like this:

Attribute Key | Change Mode | Effect Value
-|-|-
`system.skills.[abbreviation].check.minimum` | Upgrade | `[number]`
`system.skills.[abbreviation].check.maximum` | Downgrade | `[number]`

I included a `maximum` as well because if you're adding a minimum, it makes sense to add the inverse, though I can't imagine how it would be used.

When a skill check is rolled, it will check for that and either do `max(forcedMinimumValue, rolledValue)` or `max(forcedMaximumValue, rolledValue)`

**Comment by arbron** [comment]
Duplicate of https://github.com/foundryvtt/dnd5e/issues/1350, there is already a PR for this but it is blocked by the roll refactor
